projects/BiFPN/BiFPN/bifpn.py

```diff
@@ -0,0 +1,296 @@
+from typing import List
```

Comment:
The file structure can be 
- projects/BiFPN
    - bifpn
        - bifpn.py
        - utils.py